### PR TITLE
add a new router: FastRoute;

### PR DESCRIPTION
**Pull request by wujs0226**
It will cal the route path that ground station will NOT be an immediate node. **Please pass this pull request first!**

### CODE REVIEW

File: router.py [hunk 1]
```diff
@@ -97,3 +106,119 @@ def cal_n(self):
...
+            exit(0)
+
+        self.gs_neighbour_matrix = []
+        self.sat_neighbour_matrix = []
+        self.sat_distance = []
+        self.gs_sat_distance = []
+        gs_num = len(self.gs_no_list)
+        sat_num = len(self.neighbour_matrix) - gs_num
+        for i in range(len(self.neighbour_matrix)):
+            if i in self.gs_no_list:  # 地面站
```
Comment (hs-zhuwenjun) [inline router.py:141]: in english

File: router.py [hunk 2]
```diff
@@ -97,3 +106,119 @@ def cal_n(self):
...
+                self.gs_neighbour_matrix.append(self.neighbour_matrix[i])  # 加入邻接矩阵（卫星）
```
Comment (hs-zhuwenjun) [inline router.py:142]: same as above

Comment (wujs0226) [reply router.py]: solved

**Review by OreoFroyo — APPROVED** [review]
Good

**Pull request merged**